src/DiskIO/IpcIo/IpcIoFile.h

```diff
@@ -148,8 +150,12 @@ class IpcIoFile: public DiskFile
 
     static const double Timeout; ///< timeout value in seconds
 
-    typedef std::list<Pointer> IpcIoFileList;
-    static IpcIoFileList WaitingForOpen; ///< pending open requests
+    typedef std::multimap<timeval, Pointer, std::less<timeval> > WaitingFiles;
```

Comment:
Please add an allocator in case this code gets reused for something other than Rock diskers (e.g., for lots of individual files).

Reply:
... but to the wrong map: This PR does not modify RequestMap so we should not touch it without a good reason. Please add the allocator to the WaitingFiles instead. When you do, please keep in mind the "missing `const`" bug recently fixed in 2081cef for another map allocator.

Please use `using X = Y` style for new/changed type definitions.